packages/connect-examples/electron-main-process/src/electron.js

```diff
@@ -12,6 +12,7 @@ const init = () => {
         height: 775,
         webPreferences: {
             preload: path.join(__dirname, 'preload.js'),
+            sandbox: false,
```

Comment:
could you please add a comment why this is needed to be disabled?

Reply:
1265c4fe5c1643beaf257fd15f0791a79a654cf5

Reply:
<img width="560" alt="Screenshot 2022-11-22 at 13 48 46" src="https://user-images.githubusercontent.com/33235762/203319187-e42f8bac-8b73-439b-bb02-47f6e9a10783.png">

Reply:
thanks